govuk_chat_evaluation/question_router/evaluate.py

```diff
@@ -178,12 +178,16 @@ def evaluate_and_output_results(output_dir: Path, evaluation_data_path: Path):
         aggregate_results.classification_labels,
     )
 
-    write_csv_results(
-        output_dir,
-        aggregate_results.miscategorised_cases(),
-        filename="miscategorised_cases.csv",
-        data_label="miscategorised_cases",
-    )
+    if aggregate_results.miscategorised_cases():
+        write_csv_results(
+            output_dir,
+            aggregate_results.miscategorised_cases(),
+            filename="miscategorised_cases.csv",
+            data_label="miscategorised_cases",
+        )
+
+    else:
+        logging.info("\nThere are no miscategorised cases to write to file.")
```

Comment:
I don't think we need the new line at the start of this but I know we have it all over the codebase, I'm going to have a quick look and see if the others are actually needed and may remove in a separate PR

Reply:
I've had a look, we should remove the new line from this one


```suggestion
        logging.info("There are no miscategorised cases to write to file.")
```

Others seem to be used just for spacing key messages out but we wouldn't want to space this message out.